### DISCUSSION TITLE
docker run command not working.

### DISCUSSION THREAD

**Discussion by AquaCobell** [itzg/docker-minecraft-server#1712] [Q&A]
I'm trying to run a 1.16.5 paper server with the following docker command:

docker run -d -v /home/docker/mc:/data \
    -e TYPE=PAPER \
	-e PAPER_DOWNLOAD_URL="https://drive.google.com/uc?export=download&id=1JGQ0ixkKYE8de2tdKzPmQwtuwBZDmsex" \ 
    -e UID=1234 \
    -e GID=1234 \
    -e MEMORY=2G\
    -p 25565:25565 -e EULA=TRUE --name mc3 itzg/minecraft-server

but docker doesn't recognize. can someone help me?

**Comment by AquaCobell** [comment]
```
docker run -d -v /home/docker/mc:/data \
    -e TYPE=PAPER \
	-e PAPER_DOWNLOAD_URL="https://drive.google.com/uc?export=download&id=1JGQ0ixkKYE8de2tdKzPmQwtuwBZDmsex" \ 
    -e UID=1234 \
    -e GID=1234 \
    -e MEMORY=2G\
    -p 25565:25565 -e EULA=TRUE --name mc3 itzg/minecraft-server
```

**Comment by itzg** [comment]
What do the logs say?
